canary/mqtt5/canary.ts

```diff
@@ -90,8 +139,8 @@ async function doUnsubscribe(context : CanaryContext) {
     try {
         context.mqttStats.unsubscribesAttempted++;
 
-        await context.client.unsubscribe({
-            topicFilters: [ topicFilter ]
+        await getRandomClient(context.clients).unsubscribe({
```

Comment:
We have a check above in this function that subscription exists. With multiple clients this logic seems not working anymore